libnmstate/schema.py

```diff
@@ -348,3 +349,8 @@ class Neighbors:
         TLV_TYPE = "type"
         TLV_SUBTYPE = "subtype"
         ORGANIZATION_CODE = "oui"
+
+class VRF:
+    CONFIG_SUBTREE = "vrf"
+    PORT_SUBTREE = "ports"
```

Comment:
We have currently a mixture of "port" (bridge) and "ports" (team), perhaps we should sync them all to be consistent.

I do not really know which is better, plural or singular, there are some conventions floating around. Maybe just check with some other projects that use xml/yaml/json for the configuration. Ansible network unified ones?

Reply:
@ffmancera Can you also include this into your `inclusive language plan`?

Reply:
I have changed to `port` to align with LinuxBridge and OvsBridge.

Since there might no nmstate consumer using team, so we will deprecate the use of `ports` in team: https://github.com/nmstate/nmstate/issues/1328